Clava-JS/src-api/LegacyIntegrationTests - Issues.test.ts

```diff
@@ -0,0 +1,23 @@
+import { ClavaLegacyTester } from "../jest/ClavaLegacyTester.js";
+import ClavaJavaTypes from "@specs-feup/clava/api/clava/ClavaJavaTypes.js";
+import path from "path";
+
+/* eslint-disable jest/expect-expect */
+describe("IssuesTest", () => {
```

Comment:
Are all the tests here? For instance, issue_187 seems to be missing

Reply:
This file is for legacy tests only.
Since issue_187 and 190 were new and were also small tests I placed them into `Clava-JS/src-api/Issues.test.ts` when I merged `staging` into this branch.